src/test/java/deku/DekuExceptionTest.java

```diff
@@ -0,0 +1,25 @@
+package deku;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class DekuExceptionTest {
+    @Test
+    public void ExceptionTest_emptyException_String() {
+        assertEquals("AUUUUUGH! ",

```

Comment:
Good format but perhaps change String to the expected behaviour such that the naming follows recommended guideline:
featureUnderTest_testScenario_expectedBehavior();